dev/src/Helpers.ts

```diff
@@ -363,3 +361,237 @@ export const sortEnrollments = (enrollments: Enrollment[], sortBy: EnrollmentSor
         return enrollmentCompare(a, b, sortBy, descending)
     })
 }
+
+// Class with converter functions for the different proto types
+// TODO(jostein): Move this to a separate file
+// TODO(jostein): Currently used in `actions`. Consider moving use to `GRPCManager`.
+export class ProtoConverter {
+
+    /**
+     * create creates a new object of the given type
+     * @param type the type you wish to create as T.AsObject
+     * @returns the given type as a T.AsObject
+     * @example create<User.AsObject>(User) // returns a new User.AsObject
+     */
+    public static create<T>(type: (new () => any)): T {
+        return (new type()).toObject() as T
+    }
+
+    /** The following functions will convert various kinds of objects
+     *  into their respective protobuf message types */
+    // TODO(jostein): Add converting functions for all types
+    // TODO(jostein): It would be awesome to make this generic. Not sure how to do that though.
+    public static toUser = (obj: User.AsObject): User => {
```

Comment:
Seems like these conversion funcs are not used yet. Might be worth to think about if we could generate this stuff. The simple fields should be easy; looks a bit more difficult for lists and maps etc.